staging/src/k8s.io/apimachinery/pkg/runtime/converter.go

```diff
@@ -706,20 +706,18 @@ func mapToUnstructured(sv, dv reflect.Value) error {
 	if dt.Kind() == reflect.Interface && dv.NumMethod() == 0 {
 		if st.Key().Kind() == reflect.String {
 			switch st.Elem().Kind() {
-			// TODO It should be possible to reuse the slice for primitive types.
-			// However, it is panicing in the following form.
```

Comment:
Hmm - it says it was panicing in the past and you're just uncommenting it.
Can you describe what has changed that it works now?

Reply:
Sure, I changed `sv.Set(sv)` to `dv.Set(sv)`, I think this could be a typo.

Reply:
Heh...